README.md

```diff
@@ -7,6 +7,16 @@ Mozilla's internal phonebook app.
 
 *Note:* The app requires LDAP server access. You probably need a [Mozilla-internal VPN connection](https://mana.mozilla.org/wiki/display/SD/VPN) up and running. Without, you can't currently develop on *Phonebook*. You can download your VPN certificate at [login.mozilla.com](https://login.mozilla.com).
 
+## Apache, Authentication, and LDAP
+
+Previous versions of the app authenticated twice - once to the Apache (`mod_authnz_ldap`), and then again to PHP itself. This is not compatible with more complex auth methods (`mod_auth_mellon`), which set `REMOTE_USER` without providing a password for LDAP use.
```

Comment:
This is a nitpick but I think you should just write "once to Apache" instead of "once to the Apache".